poms/bom/pom.xml

```diff
@@ -48,10 +48,7 @@
                 <scope>import</scope>
             </dependency>
 
-            <!-- Sort using cd extensions && mvn groovy:execute@sort-poms -->
-            <!-- Dependencies a..z -->
```

Comment:
Please do not remove the above two lines. The second one is important for the `sort-poms` script. 

Also please remove the once removed `<!-- Camel (in alphabetical order) -->` line and re-run `cd extensions && mvn groovy:execute@sort-poms`.

Reply:
I hope that you rethink this pattern, but I will try.

Reply:
Sorry for the inconvenience, but these POMs will gradually contain hundreds of dependencies so sorting them programmatically to be able to figure out what is in there is a must.